web/packages/teleterm/src/ui/TopBar/Connections/ConnectionsIcon/ConnectionsIconStatusIndicator.tsx

```diff
@@ -36,9 +34,28 @@ const StyledStatus = styled(Box)<InternalProps>`
   border-radius: 50%;
   box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
   ${props => {
-    const { $connected, theme } = props;
-    const backgroundColor = $connected ? theme.colors.success.main : null;
-    const border = $connected
+    const { status, theme } = props;
+
+    if (status === 'warning') {
+      return css`
+        color: ${theme.colors.interactive.solid.alert.default};
+        &:after {
+          content: '⚠';
+          font-size: 12px;
```

Comment:
I know that Kenny requested that we make those icons bigger, but I just haven't had time to address this. We might come back to this in the future, since the indicators in the top bar are misaligned anyway (Connections vs Connect My Computer vs I think warning in the Identity selector).